relay-server/src/metrics_extraction/transactions.rs

```diff
@@ -457,26 +447,23 @@ mod tests {
 
         assert_eq!(metrics.len(), 5, "{:?}", metrics);
 
-        assert_eq!(metrics[0].name, "sentry.transactions.measurements.foo");
-        assert_eq!(metrics[1].name, "sentry.transactions.measurements.lcp");
+        assert_eq!(metrics[0].name, "d:transactions/measurements.foo@");
+        assert_eq!(metrics[1].name, "d:transactions/measurements.lcp@");
         assert_eq!(
             metrics[2].name,
-            "sentry.transactions.breakdowns.span_ops.ops.react.mount"
+            "d:transactions/breakdowns.ops.react.mount@"
```

Comment:
Are we sure we can omit `span_ops` here? I lack context on why this namespace was introduced, but I suspect it was to allow for different types of breakdowns in the future? There is a particular breakdown measurement called `total.time`, this would now become `d:transactions/breakdowns.total.time@ms`.

https://github.com/getsentry/relay/blob/4b7bfc0c38c0e78056054e8b5d4b66c78c4798f5/relay-general/src/store/normalize/breakdowns.rs#L190

Reply:
Good point, thank you for raising this. 

The name of breakdowns is custom per breakdown type. The ones for span ops receive an `ops.` prefix here:
https://github.com/getsentry/relay/blob/7ee898dfce656279dded87dbde1baea182ffdc3e/relay-general/src/store/normalize/breakdowns.rs#L181

This makes the `span_ops` prefix redundant. We can also remove that other prefix and keep the `span_ops`, it's just that both are not necessary. 

@dashed @untitaker, what's your take on this?

Reply:
yes `ops.` was iirc intentionally added to avoid conflicts with `total.time`

Reply:
So that means we should bring back the `span_ops` prefix, right? Is this the right mental model?

```
                           ┌──ops.*
              ┌──span_ops──┤
              │            └──total.time
   breakdowns─┤
              │
              └──other breakdowns
```

Reply:
So that means we should bring back the `span_ops` prefix, right? Is this the right mental model?

```
                           ┌──ops.*
              ┌──span_ops──┤
              │            └──total.time
   breakdowns─┤
              │
              └──other breakdowns
```

Reply:
Just confirming that `total.time` shouldn't be changed instead?